manuskript/models/abstractModel.py

```diff
@@ -122,6 +131,9 @@ def parent(self, index=QModelIndex()):
         if parentItem == self.rootItem:
```

Comment:
`if parentItem is None or parentItem == self.rootItem:` instead of duplicating the 'return qModelIndex'

Reply:
Will change. I used to have debug code inbetween and I didn't realize I could combine them when cleaning up. Also good catch on the 'is None'; I always make that mistake.

Reply:
Good! I like the optimization, and I hadn't realized "is None" acted different from "== None", I just am used to using "is None". Good that you caught the difference :)